lms/services/file.py

```diff
@@ -21,6 +21,26 @@ def get(self, lms_id, type_):
             .one_or_none()
         )
 
+    def find_copied_file(self, new_course_id, original_file: File):
```

Comment:
Not part of the plugin but is used by it. Small and straightforward enough to include here.